Formula/kubevious.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Kubevious < Formula
+  desc "Detects and prevents Kubernetes misconfigurations and violations"
+  homepage "https://github.com/kubevious/kubevious"
+  url "https://registry.npmjs.org/kubevious/-/kubevious-1.0.20.tgz"
+  sha256 "04331ac58518ce646e039203ee3c825e5b1f9e326df0dee9c7a6df7335650304"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://registry.npmjs.org/kubevious/latest"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
+  end
+
+  depends_on "node@14"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    (bin/"kubevious").write_env_script libexec/"bin/kubevious", PATH: "#{Formula["node@14"].opt_bin}:$PATH"
+  end
+
+  test do
+    assert_match "v1.0.20",
+      shell_output("#{bin}/kubevious --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Done. Please check:
https://github.com/Homebrew/homebrew-core/pull/116948/commits/c8af93c184c9941a8f00f5db9dd7aebfb28ef931#diff-93a32ca34cc401e74c3441d9604fad088e672670671dcf94e0faac118c14c9edR26-R103